rosetta/retriever/retriever.go

```diff
@@ -323,6 +323,32 @@ func (r *Retriever) Transaction(rosBlockID identifier.Block, rosTxID identifier.
 	return &transaction, nil
 }
 
+func (r *Retriever) Sequence(rosBlockID identifier.Block, rosAccountID identifier.Account, index int) (uint64, error) {
```

Comment:
Don't forget to add unit tests for this new method.

Reply:
https://github.com/optakt/flow-dps/issues/390